lib/bolt/config/modulepath.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'bolt/config'
+
+module Bolt
+  class Config
+    class Modulepath
+      BOLTLIB_PATH = File.expand_path('../../../bolt-modules', __dir__)
+      MODULES_PATH = File.expand_path('../../../modules', __dir__)
+
+      # The user_modulepath only includes the original
+      # modulepath, and is used during pluginsync
+      # since we don't want to pluginsync any of the
+      # content from BOLT_MODULES since that content includes
+      # core modules that can conflict with modules installed
+      # with an agent
```

Comment:
```suggestion
      # The user_modulepath only includes the original modulepath and is used during pluginsync.
      # We don't want to pluginsync any of the content from BOLT_MODULES since that content 
      # includes core modules that can conflict with modules installed with an agent.
```

Just some punctuation to make this comment a little easier to understand.